ncm-filesystems/src/main/perl/filesystems.pod

```diff
@@ -12,33 +11,23 @@ to create and remove blockdevices without restarting or
 re-installing.
 
 These filesystems will be later mounted/unmounted, and added/removed
-from fstab, using ncm-fstab.
+from C<< /etc/fstab >>, using ncm-fstab.
 
 The component doesn't provide any special resources at the moment. It
-just watches for changes on /system/filesystems and /system/blockdevices and
-creates new filesystems, if needed.
+just watches for changes on C<< /system/filesystems >> and C<< /system/blockdevices >>
+and creates new filesystems, if needed.
 
 A blockdevice is useful only for its ability to hold a
 filesystem. Blockdevices with no filesystems associated will not be
 created. If you want any such device, create a filesystem with
-"type"="none" and "mount"=false.
-
-=head2 To do list
-
-It will also remove filesystems and blockdevices that are not listed
```

Comment:
Good point, I'll reinsert it as a warning note. I can insert the GH issue as well if you want but I don't think documentation is the correct place for that.